dandi/tests/data/dandiarchive-docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+
+  mongo:
+    image: mongo:latest
+    volumes:
+      - dandi_girder_mongo_db:/data/db
+
+  girder:
+    image: dandiarchive/dandiarchive-girder
+    volumes:
+      - dandi_girder_assetstore:/home/assetstore
+    depends_on:
+      - mongo
+    ports:
+      - "8080:8080"
+
+  client:
+    image: dandiarchive/dandiarchive-client
+    depends_on:
+      - girder
+    ports:
+      - "8085:80"
```

Comment:
The client container isn't needed for the tests, is it?  Because changing the port numbers makes it unable to load.  (I tried configuring the container to connect to girder on port 8081 by setting the `VUE_APP_API_ROOT` environment variable on it, but that didn't work.)

Reply:
Good point! I do not think we indeed need anything from web UI client.  Let's indeed get rid of it here - should also speed up testing startup/teardown time I guess.

Note: made a new https://github.com/dandi/dandi-cli/issues/164 to tackle after this one.